Requires MCF docker network to run

> Task :svc-auth:dockerRun FAILED
docker: Error response from daemon: network mcf_default not found.


FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':svc-auth:dockerRun'.
> Process 'command 'docker'' finished with non-zero exit value 125

--- a/build.gradle
+++ b/build.gradle
@@ -19,7 +19,7 @@ buildscript {
     }
 }
 
-group = 'mcf'
+group = 'gdsace'
 
 apply plugin: 'kotlin'
 apply plugin: 'kotlin-spring'
@@ -40,7 +40,7 @@ docker {
 
 dockerRun {
     name 'svc-auth'
-    image 'mcf/svc-auth:latest'
+    image 'gdsace/svc-auth:latest'
     volumes 'src/main/resources/certs': '/app/shared/certs'
     ports '8000:8000'
     env ([
@@ -49,7 +49,6 @@ dockerRun {
     ])
     daemonize false
     clean true
-    network 'mcf_default'
 }
 
 detekt {

